tecken/tests/test_upload.py

```diff
@@ -51,21 +46,11 @@ def _join(x):
 DUPLICATED_DIFFERENT_SIZE_ZIP_FILE = _join("duplicated-different-size.zip")
 
 
-class FakeUser:
```

Comment:
This is a really weird class. I dropped this in favor of fixtures that create users in the db with the required properties.